wandb/util.py

```diff
@@ -189,6 +189,14 @@ def load(self) -> None:
             self.module.__spec__.loader.exec_module(self.module)
             self.module.__class__ = types.ModuleType
 
+            # Set the submodule as an attribute on the parent module
+            # This enables access to the submodule via normal attribute access.
+            parent = self.module.__name__.rpartition(".")[0]
+            if parent:
+                parent_module = sys.modules[parent]
+                child = self.module.__name__.rpartition(".")[-1]
```

Comment:
Instead of partitioning twice, store the result of the first `rpartition` call in a variable:

```
parent, _, child = rpartition(...)
```